lib/client/api_test.go

```diff
@@ -500,3 +498,61 @@ func TestApplyProxySettings(t *testing.T) {
 		})
 	}
 }
+
+func TestParseSearchKeywords(t *testing.T) {
+	t.Parallel()
+
+	expected := [][]string{
+		{},
+		{"foo"},
+		{"foo,bar", "some phrase's", "baz=qux's", "some other  phrase", "another one"},
+		{"服务器环境=测试,操作系统类别", "Linux", "机房=华北"},
+	}
+
+	testCases := []struct {
+		name      string
+		delimiter rune
+		specs     []string
+	}{
+		{
+			name:      "with comma delimiter",
+			delimiter: ',',
+			specs: []string{
+				"",
+				"foo",
+				`"foo,bar","some phrase's",baz=qux's ,"some other  phrase"," another one  "`,
+				`"服务器环境=测试,操作系统类别", Linux , 机房=华北 `,
+			},
+		},
+		{
+			name: "with 0 value delimiter (fallback to comma)",
+			specs: []string{
+				"",
+				"foo",
+				`"foo,bar","some phrase's",baz=qux's ,"some other  phrase"," another one  "`,
+				`"服务器环境=测试,操作系统类别", Linux , 机房=华北 `,
+			},
+		},
+		{
+			name:      "with space delimiter",
+			delimiter: ' ',
+			specs: []string{
+				"",
+				"foo",
+				`foo,bar "some phrase's" baz=qux's "some other  phrase" " another one  "`,
+				`服务器环境=测试,操作系统类别 Linux  机房=华北 `,
+			},
+		},
+	}
+
+	for _, tc := range testCases {
+		tc := tc
+		t.Run(tc.name, func(t *testing.T) {
+			t.Parallel()
```

Comment:
I would remove the parallel here, and then there's no need for the `tc := tc` line.

These tests will be super fast (they're pure computation, no network calls or anything that can block), so no need to further parallelize them.